src/zos_files/zowe/zos_files_for_zowe_sdk/datasets.py

```diff
@@ -11,13 +11,216 @@
 """
 
 import os
+from typing import Optional
 
 from zowe.core_for_zowe_sdk import SdkApi
 from zowe.core_for_zowe_sdk.exceptions import FileNotFound
 from zowe.zos_files_for_zowe_sdk.constants import FileType, zos_file_constants
 
 _ZOWE_FILES_DEFAULT_ENCODING = zos_file_constants["ZoweFilesDefaultEncoding"]
 
+
+class DatasetOption:
+
+    def __init__(
+        self,
+        like: str = None,
```

Comment:
@t1m0thyj does it make sense to have `like` as a required parameter? It seems to me like there would be times when a user would not want to base a new data set on an existing data set.

Reply:
I don't think `like` is required here since there is a default value of `None`. However I agree we should probably change the type annotation to be consistent and say `Optional[str]`